go.mod

```diff
@@ -19,5 +19,6 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/whyrusleeping/cbor-gen v0.0.0-20200812213548-958ddffe352c
 	github.com/xorcare/golden v0.6.0
+	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
```

Comment:
This is probably leftover from an IDE auto-complete or something, but I don't think should be here.